pkg/client/gen.go

```diff
@@ -294,7 +294,7 @@ func e2eManifest(cfg *GenConfig) *manifest.Manifest {
 			PluginName:   "e2e",
 			Driver:       "Job",
 			ResultType:   "e2e",
-			ResultFormat: "junit",
+			ResultFormat: "e2e",
```

Comment:
This should go back to junit, but it really doesnt matter: they get processed on the same path.